script/load_templates

```diff
@@ -131,10 +132,10 @@ my @tables = (qw(Machines TestSuites Products JobTemplates JobGroups));
 sub print_error {
     my ($res) = @_;
     die 'unknown error code - host ' . $url->host . ' unreachable?' unless $res->code;
-    printf STDERR "ERROR: %s - %s\n", $res->code, $res->message // 'no error message';
-    if ($res->body) {
-        dd($res->json || $res->body);
-    }
+    die $res->message . ': ' . join("\n", @{$res->json->{error}})
```

Comment:
Should we check if $res->message is not nil?

Reply:
We should. The documentation has a good example for printing error messages of the transaction: https://mojolicious.org/perldoc/Mojo/Transaction#error

Here we also want to show `@{$res->json->{error}}` but only if it is actually present (and it might now).